docs/sphinx/about/whats-new.txt

```diff
@@ -1,13 +1,14 @@
 Version history
 ===============
 
-5.2.0-m4 (2016 June 24)
+5.2.0-m5 (2016 July 7)
 -----------------------
 
 Top-level Bio-Formats API changes:
 
 * Java 1.7 is now the minimum supported version
 * the native-lib-loader dependency has been bumped to version 2.1.4
+* the xalan dependency has been bumped to version 2.7.2
```

Comment:
Yes, old Fiji installations which originally shipped with Java 6, and have not been updated to use Java 8 instead, will now be "dead ended" at the final Bio-Formats 5.1.x release, since that is what is currently available from the Fiji update site.

However, current (since early 2016) downloads of Fiji ship with a Java 8 JRE, and the Java-8 update site enabled by default. So these new installations will receive Bio-Formats 5.2.0 automatically.

See also http://imagej.net/2016-05-10_-_ImageJ_HOWTO_-_Java_8,_Java_6,_Java_3D

Eventually, the plan is to transition everything from Java-8 back onto the ImageJ and Fiji update sites. The prerequisite for doing that is to update the launch mechanism to use a minimal Java-6-compatible stub, which checks the Java version, and complains if it is too old. Once that is in place, we can switch everything back, so that old installations report the need to upgrade to the Java version rather than silently failing to start.